### DISCUSSION TITLE
How to set nerd fonts to bash terminal using dotfiles?

### DISCUSSION THREAD

**Discussion by 7c78** [starship/starship#5174] [Q&A]
Hi. I installed Fira nerd font and saw the font in my terminal's Preference>Appearance. But how can I set the font using dotfiles? I am using Fedora 38 xfce. Thanks

**Comment by mickimnet** [comment]
Normally, font settings are done in the terminal and the shell uses it. What is your problem exactly?

**Reply by 7c78** [reply]
I know that I can set fonts in the terminal. What I want is to store this configuration in a dotfile. So, if I switch to another computer, I just need to override all of my dotfiles and do not need to remember to set the font manually.

Edit: I am using Nix home-manager to automatically install nerd fonts and starship.

**Reply by mickimnet** [reply]
That depends on the terminal application, you're using, and if it uses configuration files. Maybe check in [NixOS options](https://search.nixos.org/options?) if there are options for your terminal available, or for the same in _Home Manager_.

**Reply by 7c78** [reply]
Thanks.
